### PR TITLE
issue #26: add support to rbac roles

### PR DESCRIPTION
**Pull request by jeremiasroma**
Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to <repo_name>?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the CONTRIBUTING.md doc.
- [x] I have added necessary documentation (if appropriate).
- [x] Any dependent changes have been merged and published in downstream modules.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by lzrocha — DISMISSED** [review]

File: _variables.tf [hunk 1]
```diff
@@ -78,3 +78,9 @@ variable "settings" {
   default     = {}
   description = "Additional settings which will be passed to the Helm chart values, see https://github.com/aws/eks-charts/tree/master/stable/aws-load-balancer-controller#configuration."
 }
+
+variable "roles" {
```
Comment (brunodasilvalenga) [inline _variables.tf:82]: As it's an object, will be good to declare, what is the schema of the object.

**Review by brunodasilvalenga — CHANGES_REQUESTED** [review]

File: _variables.tf (hunk 1 above)
Comment (jeremiasroma) [reply _variables.tf:82]: do you reckon to have it in the variables and in the example folder?

[examples/basic/variables.tf](https://github.com/DNXLabs/terraform-aws-eks-lb-controller/pull/28/commits/48933d42555b85b2a489e1f3561d30dedbc70c4e#diff-78e08707bea3c12dca8604fa4724dd81b18fca44e3c2385c387508339c8e050c)

Comment (brunodasilvalenga) [reply _variables.tf:82]: Thought was the module itself

Comment (jeremiasroma) [reply _variables.tf:82]: @brunodasilvalenga 

Done:

[_variables.tf](https://github.com/DNXLabs/terraform-aws-eks-lb-controller/blob/cf4069098ff26658bf8869614457705202e95022/_variables.tf#L82)

**Review by brunodasilvalenga — APPROVED** [review]

**Pull request merged**